src/Symfony/Component/PasswordHasher/Hasher/NativePasswordHasher.php

```diff
@@ -107,6 +107,6 @@ public function verify(string $hashedPassword, string $plainPassword): bool
      */
     public function needsRehash(string $hashedPassword): bool
     {
-        return password_needs_rehash($hashedPassword, $this->algo, $this->options);
+        return password_needs_rehash($hashedPassword, $this->algorithm, $this->options);
```

Comment:
Yes, `password_needs_rehash()` has a parameter that is named `$algo`, but that does not mean that we need to use the same naming, does it?

Reply:
I don't think so. Let's use true words